docs/migrations.rst

```diff
@@ -205,6 +205,84 @@ Queries can be executed with the ``execute()`` and ``query()`` methods. The
     the ``execute()`` command. This is especially important when using
     DELIMITERs during insertion of stored procedures or triggers which
     don't support DELIMITERs.
+
+Parameterised queries
+~~~~~~~~~~~~~~~~~~~~~
+
+Both the ``execute()`` and ``query()`` methods accept a second argument for an array of values for parameterised queries.
+
+The following example shows queries with positional ? placeholders.
+
+.. code-block:: php
+
+        <?php
+
+        use Phinx\Migration\AbstractMigration;
+
+        class MyNewMigration extends AbstractMigration
+        {
+            /**
+             * Migrate Up.
+             */
+            public function up()
+            {
+                $queryParams = array(10, 15);
+
+                // execute()
+                $count = $this->execute('DELETE FROM users WHERE user_id BETWEEN ? AND ?', $queryParams);
+
+                // query()
+                $rows = $this->query('SELECT * FROM users WHERE user_id BETWEEN ? AND ?', $queryParams);
+            }
+
+            /**
+             * Migrate Down.
+             */
+            public function down()
+            {
+
+            }
+        }
+
+This example show queries with named placeholders and an optional bind type.
+
+.. code-block:: php
+
+        <?php
+
+        use Phinx\Migration\AbstractMigration;
+
+        class MyNewMigration extends AbstractMigration
+        {
+            /**
+             * Migrate Up.
+             */
+            public function up()
+            {
+                $queryParams = array(
```

Comment:
I like the idea of including a bind class, but I don't think it's entirely necessary for *all* binds. Of the listed syntaxes, I think that this one is best:

```php
new QueryBind(5, QueryBind::TYPE_INT)
```

However, for scalar types, I don't think it's entirely necessary. It should be possible to simply have:

```php
$bindValues = array(
    'a' => 'foo',
    'b' => 15
);

And have this automatically be recognized as `TYPE_STRING` and `TYPE_INT`. I see the QueryBind being most useful when dealing with complex data. With simple data, it needs to get out of the way or it will be cumbersome.